### PR TITLE
Some basic factors that affects the performance of system using deep learning 

### LINKED ISSUE #3630: Some basic factors that affects the performance of system using deep learning
Proposed title of article

Some fundamental factors that have an impact on the performance of deep learning systems. 

Introduction paragraph (2-3 paragraphs):

The performance of different optimisers in deep learning is highly dependent on the problem under consideration and the available tuning budget.

Tuning the hyperparameters of a single optimiser is nearly as successful as evaluating the default hyperparameters of many optimisers.

This could be because adaptive approaches have an implicit learning rate schedule (e.g., Adam).

Adding an unturned learning rate schedule enhances performance on average, but there is a lot of variation.

Adam continues to provide a solid foundation.

Newer methods frequently fail to surpass it, showing the value of specialized optimisers.

Among these are:

·       Adjusting the data budget

·       This is the sort of issue that the system is supposed to solve.

·       The optimisers were chosen incorrectly.

 

This article will help the user gain a better and more practical grasp of these factors, as well as how to address them.

 

Key takeaways:

What factors influence the deep learning systems' performance?

What are the steps to achieving a system's ideal performance rate?

How can a user tell if the performance of his or her system is satisfactory?

 

References:

 

N/A

 

Templates to use as

### PR DESCRIPTION
**Pull request by KayTee24**
### Article checklist

> If you have not already, please go over our [Resources Page](https://github.com/section-engineering-education/engineering-education/blob/master/new_contributors/resources-page.md) for more tips and tricks on how to improve your overall technical writing so reviews are swifter, increase the chance of a payout and provide more value to our readers.

> For first-time contributors and for more details on our submission guidelines, **see our [Contributing Guidelines](https://github.com/section-engineering-education/engineering-education/blob/master/new_contributors/CONTRIBUTING.md)**.

#### Formatting and structure checks
- [x] Have you used the correct folder and file structure? - **see [Contribution Guidelines](https://github.com/section-engineering-education/engineering-education/blob/master/new_contributors/CONTRIBUTING.md)**
- [x] Is your article properly formatted in Markdown? - **see [Github Markdown guide](https://github.com/adam-p/markdown-here/wiki/Markdown-Cheatsheet)**
- [x] Have you used our preferred heading structure? - **[H3 (###) and up](https://github.com/adam-p/markdown-here/wiki/Markdown-Cheatsheet#headers)**
- [x] Does your article follow any of the suggested structure formats? - **see [example formats](https://github.com/section-io/engineering-education/blob/master/CONTRIBUTING.md#example-format-structure)**

- [x] Is your article over 750 words? - **Tool**: [Wordcounter.net](https://wordcounter.net)
- [x] Is the article you are submitting an in-depth and unique article? Does it go beyond what is in the official docs and what is covered in other blog sites. See these articles as [examples](https://github.com/section-engineering-education/engineering-education/blob/master/new_contributors/resources-page.md).
- [x] Does your article provide enough value and detail about your topic? - **Articles should be clear, accurate, and fully explained.**
- [x] Can your article be understood by beginners? - **Assume the audience is smart but has no prior exposure to the common terminology in your article.**

- [x] Have you included a hero image that is 600x400 pixels and have the copyright to use it? - **Tools:** [Pixlr Image Editor](https://pixlr.com/e) for resizing and [Unsplash](https://unsplash.com) for Creative Commons images
      
#### Grammar & plagiarism checks
- [x] Have you spellchecked and grammar checked your article? - **Tools:** [Write&Improve](https://writeandimprove.com/?lang=en-GB) - [ProWritingAid](https://prowritingaid.com/) - [Grammarly](https://grammarly.com) 
- [x] Please place your article through a 3rd party plagiarism checker. We suggest using Quetext, [this tool](https://www.quetext.com/) is free and has a daily limit. **We typically accept articles with 10% or less.**
- [x] Have you checked your article for readability? - **Tool:** [Hemmingway](http://www.hemingwayapp.com)
- [x] Have you added sources for quotes and images that aren't yours?

#### Technical checks
- [ ] Are your code snippets properly formatted for syntax highlighting - **see [Syntax guide](https://gohugo.io/content-management/syntax-highlighting)**
- [ ] Have you checked your code runs correctly and you've highlighted all necessary dependencies for installation?
- [ ] Are the software programs and packages you're highlighting in your article up to date, using current versions and not deprecated?

This closes #3630

> Finally, delete the article checklist notes in blockquotes and submit your PR. We look forward to reviewing your article.

### CODE REVIEW

**Comment by KayTee24** [comment]
Some factors that affects the performance of systems using deep learning #3630
